src/components/DayPickerNavigation.jsx

```diff
@@ -106,6 +115,13 @@ export default function DayPickerNavigation(props) {
       >
         {navNextIcon}
       </button>
+    )
+  };
+
+  return ( // sag1v rtl
+    <div className={navClassNames}>
+      {!isVerticalScrollable && !isRTL ? prevButton() : nextButton()}
+      {!isRTL ? nextButton() : prevButton()}
```

Comment:
instead of making these functions that are only executed conditionally, let's use the typical jsx pattern if `isRTL && nextButtonElements` etc.